cypress/integration/premium/dashboard.spec.ts

```diff
@@ -0,0 +1,49 @@
+import TeamSettingsPage from "../../pages/teamSettingsPage";
+
+describe("Dashboard", () => {
+  before(() => {
+    Cypress.session.clearAllSavedSessions();
+    cy.setup();
+    cy.loginWithCySession();
+    cy.viewport(1200, 660);
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  describe("Activity Card", () => {
+    beforeEach(() => {
+      cy.loginWithCySession();
+    });
+
+    it("displays activity when editing a teams agent options", () => {
+      cy.visit("/settings/teams");
+
+      cy.getAttached(".no-teams").within(() => {
+        cy.getAttached(".no-teams__inner-text").within(() => {
+          cy.contains("button", /create team/i).click();
+        });
+      });
+
+      cy.findByRole("textbox", { name: "Team name" }).type("Team 1");
+      cy.findByRole("button", { name: "Create" }).click();
+
+      // we've just created this team so we know we only have one team with id = 1
+      TeamSettingsPage.visitTeamAgentOptions(1);
+      TeamSettingsPage.editAgentOptionsForm("test:");
```

Comment:
Same here, may need to wait until the API responds with success before going to check for the activity.